export_monitor/export_monitor.R

```diff
@@ -17,46 +17,44 @@ if (any(!check)) {
 #data up to 2017 exists
 
 cbsa_export <- readxl::read_xlsx("V:/Export Monitor/2018/Deliverables/Deliverables/Metros Data/Metros by Total, NAICS 2 3.xlsx", sheet = "Total") %>%
-  filter(Year == 2017) %>%
-  select(-c(2,13:16))
+  filter(Year == 2017)
+
 
 #use old column names as labels (use View() to see labels)
 set_label(cbsa_export)<-colnames(cbsa_export)
```

Comment:
old column names were assigned as labels before any column deletions or renaming occurs. At the point when column renaming and deletions begin (see below), each column has already been labelled, so removing or renaming columns will not 'break' the labelling structure or lead to mislabelling